x-pack/test_serverless/functional/test_suites/common/examples/search_examples/index.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+// TODO: Changed from PluginFunctionalProviderContext to FtrProviderContext in Serverless
+import type { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getService, loadTestFile }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const kibanaServer = getService('kibanaServer');
+
+  describe('search examples', function () {
+    before(async () => {
+      // TODO: This fails in Serverless with
+      // "index_not_found_exception: no such index [.kibana_ingest]",
+      // but the tests still run
+      try {
+        await esArchiver.emptyKibanaIndex();
+      } catch (e) {
+        // eslint-disable-next-line no-console
+        console.error(e);
+      }
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
+      ); // need at least one index pattern
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/logstash_functional');
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
+      );
+    });
+
+    // TODO: Removed `search_session_example` since
+    // search sessions are not supported in Serverless
+    loadTestFile(require.resolve('./search_example'));
+    loadTestFile(require.resolve('./search_sessions_cache'));
+    loadTestFile(require.resolve('./partial_results_example'));
+    loadTestFile(require.resolve('./sql_search_example'));
```

Comment:
I wonder if we should remove sql search example, since we will remove SQL in Discover, it's behind an Advanced Settings flag, and else it's just used in Canvas, which is not supported in serverless.

Reply:
Unfortunately for Elon I agree with you. We also probably don't need the search session tests either, guess I was tired from migrating so many tests by this point 😅 Dropped them both: https://github.com/elastic/kibana/pull/163411/commits/53569a2009c9fbb0da4ca7bcf9eaf11308e3313d.